lib/filesharing/filesharing_logic/lib/src/models/folder.dart

```diff
@@ -127,7 +127,7 @@ class Folder {
           fileSharingData: fileSharingData,
           folderName: folderName,
           folderPath: folderPath,
-          attempt: attempt++);
+          attempt: attempt + 1);
```

Comment:
```suggestion
          attempt: ++attempt;
```
This would also been possible, but I think I like `attempt + 1` better.